### ISSUE TITLE
Would like to create vanilla project from wizard

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#9]
```
I'd like to create an empty(ish) project from the wizard. With or without a 
framework, always including PhoneGap of course, but starting with a plain 
index.html.

I've attached a sample index.html. I suppose some wizardry to tuck the jQuery 
or Sencha links/script/etc into it would be needed. 


```

Original issue reported on code.google.com by `libby.ba...@gmail.com` on 20 Jan 2011 at 3:46

Attachments:
- [index.html](https://storage.googleapis.com/google-code-attachments/mobile-web-development-with-phonegap/issue-9/comment-0/index.html)

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 20 Jan 2011 at 5:53

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 20 Jan 2011 at 4:52
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 20 Jan 2011 at 4:54
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Requested by Raymond Camden 
https://groups.google.com/forum/#!topic/phonegap/ANrOHjndaZs
```

Original comment by `paul.beu...@gmail.com` on 4 Nov 2011 at 6:54
- Added labels: **Milestone-Release1.3.0**

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 17 Nov 2011 at 10:18
- Added labels: **Milestone-Release1.2.6**
- Removed labels: **Milestone-Release1.3.0**

**Comment by GoogleCodeExporter** [comment]
```
The 1.2.6 wizard will add a Create minimal PhoneGap project radio button to the 
Projec Contents section
```

Original comment by `paul.beu...@gmail.com` on 16 Dec 2011 at 9:06
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixes available with 1.2.6
```

Original comment by `paul.beu...@gmail.com` on 21 Dec 2011 at 3:57
- Changed state: **Fixed**
